extensions/apns_message_handler.go

```diff
@@ -40,44 +39,41 @@ import (
 
 var apnsResMutex sync.Mutex
 
-// Notification is the notification base struct
+// Notification is the notification base struct.
 type Notification struct {
 	DeviceToken string
 	Payload     interface{}
 	Metadata    map[string]interface{} `json:"metadata,omitempty"`
 	PushExpiry  int64                  `json:"push_expiry,omitempty"`
 }
 
-// APNSMessageHandler implements the messagehandler interface
+// APNSMessageHandler implements the messagehandler interface.
 type APNSMessageHandler struct {
+	feedbackReporters            []interfaces.FeedbackReporter
```

Comment:
where these changes done in order to reorder the members?

Reply:
Yes, one of the linters suggested it as a way to improve memory usage

Reply:
o.O, that's new to me.

Reply:
It was new to me as well.

This is the linter: https://github.com/mdempsky/maligned

And this article gives a brief explanation about the issue: https://medium.com/@sebassegros/golang-dealing-with-maligned-structs-9b77bacf4b97